### ISSUE TITLE
map - accessible name of map

### ISSUE THREAD

**Issue by IdaAkiwumi** [religious-ecologies/omeka-s-theme#73]
When using Voice Over Utility and tabbing to the map, the name of the map isn't mentioned. VOU just says "group" and from the screenshot below the screen reader only highlights a small random portion of the map (with the black rectangular box), when it should highlight/surround the entire map.

<img width="1110" alt="Screen Shot 2020-12-08 at 4 02 31 PM" src="https://user-images.githubusercontent.com/618307/101541275-3d728300-396f-11eb-8e5a-67034d941647.png">

**Comment by IdaAkiwumi** [comment]
Also, speaking of the map, based on the information here about ESRI maps: http://web-accessibility.carnegiemuseums.org/content/maps/ , is there a way to accompany this map with tabular data that can be viewed and maybe altered? I realize the Denominations and Locations pages take care of this for the most part, so a solution could be making clear on the map page that this information can also be accessed on those pages, particularly Locations page.

**Comment by kimisgold** [comment]
Would including some language directing users to the other sections of the site be adequate? "We recommend that our visually impaired and blind users explore the schedules through our text-based query builder or denomination and location filtered views."

**Comment by IdaAkiwumi** [comment]
Yes, I think that would be a good short term solution @kimisgold

**Comment by kimisgold** [comment]
Also, do you think the new page heading "Map of Schedules" sufficiently labels the map?

**Comment by IdaAkiwumi** [comment]
I think something along the lines of "Geographical Map of Denomination Schedules"  or "Geographical Map of Schedules" would be best for now since the heading and link name of the page is already "Map of Schedules".

**Comment by kimisgold** [comment]
I put in "Interface for geographic map of schedules."
